software/myip.yml

```
@@ -0,0 +1,13 @@
+name: "MyIP"
+website_url: "https://ipcheck.ing"
+source_code_url: "https://github.com/jason5ng32/MyIP"
+description: "All in one IP Toolbox. Easy to check what's your IPs, IP geolocation, check for DNS leaks, examine WebRTC connections, speed test, ping test, MTR test, check website availability and more."
+licenses:
+  - MIT
+platforms:
+  - Nodejs
```

Docker can be used to host this (https://github.com/jason5ng32/MyIP?tab=readme-ov-file#using-docker)
```suggestion
platforms:
  - Nodejs
  - Docker
```